histogram_analysis.py

```diff
@@ -4,155 +4,74 @@
 from bokeh.charts import Bar, output_file, show
 from pandas import *
 
-''' Download Texts and write to files
-
-oliver_twist_full_text = URL('http://www.gutenberg.org/ebooks/730.txt.utf-8').download()
-#write Oliver Twist to a file
-f = open('some_text_file.pickle','w')
-pickle.dump(oliver_twist_full_text, f)
-f.close()
-
-wizard_of_oz_full_text = URL('http://www.gutenberg.org/cache/epub/55/pg55.txt').download()
-#write OWizard of Oz to a file
-f = open('wizard_of_oz.pickle','w')
-pickle.dump(wizard_of_oz_full_text, f)
-f.close()
-
-hamlet_full_text = URL('http://www.gutenberg.org/cache/epub/1524/pg1524.txt').download()
-hamlet_full_text = hamlet_full_text[hamlet_full_text.find('SCENE.'):]
-f = open('hamlet.pickle','w')
-pickle.dump(hamlet_full_text, f)
-f.close()
-
-romeo_full_text = URL('http://www.gutenberg.org/cache/epub/1112/pg1112.txt').download()
-romeo_full_text = romeo_full_text[romeo_full_text.find('SCENE.'):]
-f = open('romeo.pickle','w')
-pickle.dump(romeo_full_text, f)
-f.close()
-
-lear_full_text = URL('http://www.gutenberg.org/cache/epub/1128/pg1128.txt').download()
-lear_full_text = lear_full_text[lear_full_text.find('Scene:'):]
-f = open('lear.pickle','w')
-pickle.dump(lear_full_text, f)
-f.close()
-
-caesar_full_text = URL('http://www.gutenberg.org/cache/epub/1120/pg1120.txt').download()
-caesar_full_text = caesar_full_text[caesar_full_text.find('SCENE:'):]
-f = open('caesar.pickle','w')
-pickle.dump(caesar_full_text, f)
-f.close()
 '''
-#Make string copies of books 
-#oliver_file = open('some_text_file.pickle','r')
-#copy_of_oliver_twist = pickle.load(oliver_file)
-
-#wizard_file = open('wizard_of_oz.pickle','r')
-#copy_of_wizard_of_oz = pickle.load(wizard_file)
-
-hamlet_file = open('hamlet.pickle','r')
-copy_of_hamlet = pickle.load(hamlet_file)
-
-romeo_file = open('romeo.pickle','r')
-copy_of_romeo = pickle.load(romeo_file)
-
-lear_file = open('lear.pickle','r')
-copy_of_lear = pickle.load(lear_file)
+BEFORE RUNNING: 
```

Comment:
Nice documentation here. This could also be something that might go into README.